src/users/users.php

```diff
@@ -352,30 +352,40 @@ public function get()
 	{
```

Comment:
for getting single user we should use different resource (resource=user)
https://github.com/techjoomla/plg_api_users/blob/95b2d686206a4d460a282c3a82255239e4801400/src/users/user.php#L155